Cargo.toml

```diff
@@ -7,6 +7,7 @@ license = "MIT"
 repository = "https://github.com/gnosis/ethcontract-rs"
 homepage = "https://github.com/gnosis/ethcontract-rs"
 documentation = "https://docs.rs/ethcontract"
+keywords = ["web3", "ethereum", "contract", "async"]
```

Comment:
perhaps "smart contract" is more precise? Do we want to include "solidity" as well?

Reply:
I'll add it as a separate PR later when I do some cleanup (like adding badges and getting crates.io to show a better landing page)

Reply:
Created #35